teamB/eungSooKim/app.js

```diff
@@ -53,11 +49,118 @@ app.post("/users", async (req, res, next) => {
             password
         ) VALUES (?, ?, ?);
         `,
-    [name, email, password]
+    [name, email, passWord]
   );
   res.status(201).json({ message: "userCreated" });
 });
 
+app.post("/posts/write/:userId", async (req, res) => {
+  const { title, content } = req.body;
+  const userId = req.params.userId;
```

Comment:
```suggestion
  const { title, content, userId } = req.body;
```

어떤 정보는 Request의 Body로, 다른 어떤 정보는 path parameter로 받는 이유가 따로 있을까요?

그게 아니라면 위의 코멘트를 반영한 뒤 userId는 다른 값들과 함께 body로 담아서 받을 수 있게 구현해주세요-!